### PR TITLE
feat(pci.projects.dashboard): disable discovery project creation in IN when undergoing kyc validation

### PR DESCRIPTION
**Pull request by brofik**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #TAPC-625 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/onboarding/onboarding.controller.js [hunk 1]
```diff
@@ -3,19 +3,38 @@ import illustrationUs from './assets/onboarding-us.png';
...
       name: 'PublicCloud::pci::projects::onboarding',
     });
+
+    this.myIdentitySectionLink = this.coreURLBuilder.buildURL(
+      'dedicated',
+      '#/support/tickets',
+    );
+  }
+
+  isIndianAccountUnderVerification() {
```
Comment (sachinrameshn) [inline packages/manager/modules/pci/src/projects/onboarding/onboarding.controller.js:35]: To be more generic: 
```suggestion
  isKycValidationUnderVerification() {
```

File: packages/manager/modules/pci/src/projects/onboarding/onboarding.html [hunk 2]
```diff
@@ -65,6 +65,24 @@ <h1 data-translate="pci_projects_onboarding_us_title"></h1>
     </div>
 
     <div class="d-grid gap-1 col-md-6 offset-xs-1 offset-md-3 mb-4 text-center">
+        <!-- Indian Account under verification banner -->
+        <oui-message
+            data-type="info"
+            data-ng-if="$ctrl.isIndianAccountUnderVerification()"
+        >
```
Comment (sachinrameshn) [inline packages/manager/modules/pci/src/projects/onboarding/onboarding.html:72]:
```suggestion
            data-ng-if="$ctrl. isKycValidationUnderVerification()"
        >
```

File: packages/manager/modules/pci/src/projects/onboarding/onboarding.html [hunk 3]
```diff
@@ -77,6 +95,7 @@ <h1 data-translate="pci_projects_onboarding_title"></h1>
                 data-items="$ctrl.summary.contracts"
                 data-model="$ctrl.model.agreements"
                 data-name="agreements"
+                data-disabled="$ctrl.isIndianAccountUnderVerification()"
```
Comment (sachinrameshn) [inline packages/manager/modules/pci/src/projects/onboarding/onboarding.html:98]:
```suggestion
                data-disabled="$ctrl. isKycValidationUnderVerification()"
```

File: packages/manager/modules/pci/src/projects/onboarding/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -12,5 +12,7 @@
...
   "pci_projects_onboarding_us_create_project": "Créez votre premier projet OVH Public Cloud",
   "pci_projects_onboarding_us_banner_trusted_customer_info_1": "Vous souhaitez plus d’informations ou passer commande ?",
-  "pci_projects_onboarding_us_banner_trusted_customer_info_2": "Demandez qu’un conseiller OVHcloud vous rappelle gratuitement"
+  "pci_projects_onboarding_us_banner_trusted_customer_info_2": "Demandez qu’un conseiller OVHcloud vous rappelle gratuitement",
+  "pci_projects_onboarding_banner_account_under_verification_description": "Vous souhaitez créer un projet Public Cloud : la réglementation en vigueur nécessite une vérification de votre identité. Le traitement de vos documents est actuellement en cours d'analyse par notre équipe. Pour plus d'information, vous pouvez consulter ",
```
Comment (sachinrameshn) [inline packages/manager/modules/pci/src/projects/onboarding/translations/Messages_fr_FR.json:16]:
```suggestion
  "pci_projects_onboarding_banner_kyc_validation_under_verification_description": "Vous souhaitez créer un projet Public Cloud : la réglementation en vigueur nécessite une vérification de votre identité. Le traitement de vos documents est actuellement en cours d'analyse par notre équipe. Pour plus d'information, vous pouvez consulter ",
```

File: packages/manager/modules/pci/src/projects/onboarding/translations/Messages_fr_FR.json [hunk 5]
```diff
@@ -12,5 +12,7 @@
...
+  "pci_projects_onboarding_banner_account_under_verification_link": "Mes demandes d'assistance"
```
Comment (sachinrameshn) [inline packages/manager/modules/pci/src/projects/onboarding/translations/Messages_fr_FR.json:17]:
```suggestion
  "pci_projects_onboarding_banner_kyc_validation_under_verification_link": "Mes demandes d'assistance"
```

**Review by sachinrameshn — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/onboarding/onboarding.controller.js (hunk 1 above)
Comment (brofik) [reply packages/manager/modules/pci/src/projects/onboarding/onboarding.controller.js]: Done.

File: packages/manager/modules/pci/src/projects/onboarding/onboarding.html (hunk 2 above)
Comment (brofik) [reply packages/manager/modules/pci/src/projects/onboarding/onboarding.html]: Done.

File: packages/manager/modules/pci/src/projects/onboarding/onboarding.html (hunk 3 above)
Comment (brofik) [reply packages/manager/modules/pci/src/projects/onboarding/onboarding.html]: Done.

File: packages/manager/modules/pci/src/projects/onboarding/translations/Messages_fr_FR.json (hunk 4 above)
Comment (brofik) [reply packages/manager/modules/pci/src/projects/onboarding/translations/Messages_fr_FR.json]: Done.

File: packages/manager/modules/pci/src/projects/onboarding/translations/Messages_fr_FR.json (hunk 5 above)
Comment (brofik) [reply packages/manager/modules/pci/src/projects/onboarding/translations/Messages_fr_FR.json]: Done.

**Review by sachinrameshn — DISMISSED** [review]

**Review by y4nnL — APPROVED** [review]

**Review by qpavy — APPROVED** [review]

**Review by darsene — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**